src/library/dlgtagfetcher.cpp

```diff
@@ -10,6 +10,15 @@
 
 namespace {
 
+// Percent of QProgressBar when dialog populated, "fingerprinting".
+int kPercentOfPopulation = 10;
+
+// Percent of "identifying" and "retrieving" metadata.
+int kPercentOfConstantTask = 15;
+
+// Percent left for Recording ID's found, "Fetching track data from Musicbrainz".
+int kPercentOfRecordingsFound = 60;
```

Comment:
```suggestion
// Percent of QProgressBar when dialog populated, "fingerprinting".
constexpr int kPercentOfPopulation = 10;

// Percent of "identifying" and "retrieving" metadata.
constexpr int kPercentOfConstantTask = 15;

// Percent left for Recording ID's found, "Fetching track data from Musicbrainz".
constexpr int kPercentOfRecordingsFound = 60;
```

The comment does not really match the variable names. Please include the information in the variable name and replace the comment by a brief overview how the progress is calculated. So that later authors know where to adjust the values if required.

Reply:
Done 👍 , I hope the new changes are ok.